packages/es-components/src/components/controls/AdditionalHelp.tsx

```diff
@@ -5,6 +5,12 @@ import styled from 'styled-components';
 import { useTheme } from '../util/useTheme';
 import ValidationContext from './ValidationContext';
 import Icon from '../base/icons/Icon';
+import { ValidationStyleType } from 'es-components-shared-types';
+
+interface AdditionalHelpProps {
+  children: React.ReactNode;
+  hasValidationIcon: boolean;
+}
```

Comment:
So we fully support the type, we should include the underlying JSX props as well.
```suggestion
export type AdditionalHelpProps = JSXElementProps<'div'> & {
  hasValidationIcon: boolean;
}
```